.github/workflows/release.yml

```
@@ -119,7 +125,30 @@ jobs:
+            -e WANDERER_ENV=test \
+            guarzo/wanderer-notifier:sha-${{ steps.version.outputs.short_sha }} \
+            /bin/sh -c "echo 'Testing Docker image basic functionality:' && \
+                     echo 'GLIBC Version:' && ldd --version | head -n1 && \
+                     echo 'Elixir Version:' && elixir --version && \
+                     echo 'Notifier Version:' && cat /app/VERSION && \
+                     echo 'Basic validation successful'"
+
+          echo "Docker image test passed successfully!"
+
+      - name: Push Docker image with release tags
+        uses: docker/build-push-action@v4
```

actionlint (1.7.4)

the runner of "docker/build-push-action@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)